src/modules/repository/implementation/sequelize/migrations/20241215122200-create-paranet-assets.js

```diff
@@ -0,0 +1,60 @@
+export async function up({ context: { queryInterface, Sequelize } }) {
+    await queryInterface.createTable('paranet_assets', {
+        id: {
+            type: Sequelize.INTEGER,
+            primaryKey: true,
+            autoIncrement: true,
+        },
+        blockchain_id: {
+            type: Sequelize.STRING,
+            allowNull: false,
+        },
+        ual: {
```

Comment:
I would make ual and paranet_ual as unique keys unique you can do it like this in raw SQL
```ALTER TABLE `paranet_asset` ADD UNIQUE `unique_index`(`ual`, `paranet_ual`);```

To prevent creation of duplicates

Reply:
```await queryInterface.sequelize.query(`...`);```